.teamcity/_self/projects/WebApp.kt

```diff
@@ -733,8 +733,8 @@ object CheckCodeStyleBranch : BuildType({
 	}
 })
 
-object RunCalypsoPlaywrightE2eTests : BuildType({
-	name = "Playwright E2E tests"
+object RunCalypsoPlaywrightE2eDesktopTests : BuildType({
```

Comment:
This may change the ID of the job, losing all existing history. Are we ok with that?

Reply:
Losing history now instead of later (when there is run history built up) would be preferable. This is assuming there is no way to avoid losing existing history when renaming the job.

Reply:
There is. Add `uuid = "23cc069f-59e5-4a63-a131-539fb55264e7"` here and it should preserve it.

Reply:
Added UUID in. Thanks!